src/utils.js

```diff
@@ -27,6 +27,8 @@ export const LUMIGO_EVENT_KEY = '_lumigo';
 export const STEP_FUNCTION_UID_KEY = 'step_function_uid';
 export const GET_KEY_DEPTH_ENV_KEY = 'LUMIGO_KEY_DEPTH';
 export const DEFAULT_GET_KEY_DEPTH = 3;
+export const EXECUTION_TAGS_KEY = 'lumigo_execution_tags_no_scrub';
```

Comment:
```suggestion
export const EXECUTION_TAGS_KEY = 'lumigoExecutionTags';
```
no_scrub is implementation :)

Reply:
That why I put it in a variable with a normal name :)
There are 2 reasons for this:
1. For the skip-scrubbing mechanism, we want to add only keys that will probably be unique. Also, maybe in the future we'll want to skip scrubbing for all keys that end with "_no_scrub".
2. It's already the key in the python-tracer and in the backend.